libs/zk/include/nil/crypto3/zk/commitments/detail/polynomial/proof_of_work.hpp

```diff
@@ -44,37 +45,40 @@ namespace nil {
                     using transcript_type = transcript::fiat_shamir_heuristic_sequential<transcript_hash_type>;
                     using output_type = OutType;
 
-                    static inline OutType generate(transcript_type &transcript, OutType mask=0xFFFF) {
+                    static inline std::array<std::uint8_t, sizeof(OutType)>
+                        int_be(OutType v) {
```

Comment:
Maybe rename this to something like to_byte_array() ?

Reply:
renamed